zscaler/icon_zscaler/actions/blacklist_url/action.py

```diff
@@ -25,9 +25,12 @@ def run(self, params={}):
         normalized_urls = []
         for url in urls:
             if url and not url.startswith("http"):
-                self.logger.info(f"URL did not begin with protocol, prefixing with http://{url} ...")
                 url = f"http://{url}"
-
+                self.logger.info(f"URL did not begin with protocol, prefixing with {url} ...")
```

Comment:
```suggestion
                self.logger.info(f"URL did not begin with protocol, prefixing with 'http://' ...")
```
Adjusting to prevent some confusion. The user may think their URL is being prefixed with the entire URL otherwise :)